nucliadb/nucliadb/writer/resource/basic.py

```diff
@@ -188,3 +193,13 @@ def set_processing_info(bm: BrokerMessage, processing_info: ProcessingInfo):
     if processing_info.account_seq is not None:
         bm.basic.last_account_seq = processing_info.account_seq
     bm.basic.queue = bm.basic.QueueType.Value(processing_info.queue.name)
+
+
+def validate_classifications(paragraph: ParagraphAnnotation):
+    classifications = paragraph.classifications
+    if len(classifications) == 0:
+        raise ValidationError("ensure classifications has at least 1 items")
```

Comment:
Why not just answer the HTTP exception as its on the writer ?

Reply:
Yes, that works too. 
My rationale was that `fastapi.HTTPException` is something that belongs to the "view" logic (the outermost layer of the writer), whereas the `parse_basic_modify` and `parse_basic` are more internal. I thought that having a custom `ValidationError` was making the `validate_classifications` function more reusable.